src/Components/SmartComponents/SystemCves/SystemCves.js

```diff
@@ -128,9 +128,11 @@ export const SystemCVEs = ({
         dispatch(changeSystemCVEsParameters(params));
     };
 
+    const paramsWithoutAppname = (params) => Object.fromEntries(Object.entries(params).filter((p) => p[0] !== 'appName'));
+
     useEffect(() => {
         apply(
-            isEmpty(urlParameters) ?
+            isEmpty(paramsWithoutAppname(urlParameters)) ?
                 { advisory_available: 'true' }
```

Comment:
```suggestion
                { advisory_available: 'true' , appName: 'vulnerabilities' }
```

Making this change will fix that problem.

Reply:
thanks for pointing this out, completely missed this
it should be fixed now